registrations/views.py

```diff
@@ -273,3 +275,54 @@ def post(self, request, *args, **kwargs):
         accepted = {"accepted": True}
         send_public_registration_notifications.delay()
         return Response(accepted, status=status)
+
+
+class UserDetailList(APIView):
+    """ UserDetailList Interaction
+        GET - returns a detailed list of system users
+    """
+    permission_classes = (IsAuthenticated,)
+
+    def get_data(self, page_size, offset):
+
+        def dictfetchall(cursor):
+            """Return all rows from a cursor as a dict"""
+            columns = [col[0] for col in cursor.description]
+            return [
+                dict(zip(columns, row))
+                for row in cursor.fetchall()
+            ]
+
+        sql = """
+            select *
+            from get_registrations('{}', '*', '*', '*', '*', {}, {})""".format(
+            settings.DBLINK_CONN, page_size, offset)
```

Comment:
So we do set `DBLINK_CONN` ourselves, and I know that the user input `page` is forced into an int, but I'm still concerned for maybe in future if someone else comes along and changes things, that there's a chance for sql injection.

Instead of `.format` here, could we maybe not rather use the `params` argument of `execute`, which does protect against sql injection?